week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -7,18 +7,23 @@
   NOTE: only the names should be returned, not the means of transport.
 */
 
-function journeyPlanner() {
-
+function journeyPlanner(arr, str) {
+  let stationNames = arr
+    .filter(element => element.includes(str))
+    .map(element => element[0]);
+  return stationNames;
```

Comment:
Same as above

Reply:
Thank you @ChrisOwen101 noted and refactored.